superset-frontend/packages/superset-ui-core/src/query/types/Query.ts

```diff
@@ -255,6 +255,7 @@ export type ErrorSource = 'dashboard' | 'explore' | 'sqllab' | 'crud';
 
 export type SupersetError<ExtraType = Record<string, any> | null> = {
   error_type: ErrorType;
+  errorType?: ErrorType;
```

Comment:
@justinpark Can't we fix the API responses to use the same field? Having both in the type definition looks really bad and is an indicative that we should fix the APIs.